<launch>  
<!-- <param name="use_sim_time" value="true"/> -->
    <node name="rviz" pkg="rviz" type="rviz"  args="-d $(find localizationUsingAreaGraph)/config/areaGraphRviz.rviz " required="true"/> 

    <!-- <param name="step"  value="1" /> -->
    <!-- <arg name="node_start_delay" default="2.0" />   -->
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="AGmap_tf_broadcaster" args="5 0 0 0 0 0 1 AGmap map" /> -->
    <node name="world_map" pkg="tf" type="static_transform_publisher"  args="0 0 0 0 0 0 /world /map 100"/>
    <rosparam file="$(find localizationUsingAreaGraph)/config/params.yaml" command="load" />
    <!-- 0510 AG -->
    <node pkg="tf" type="static_transform_publisher" name="camera_base_link"
                        args="9.8 -34.3  0.0 0 0 -0.649 0.760  map AGmap 100" />
    <node pkg="tf" type="static_transform_publisher" name="PandarQT_map" args="0 0 0 0 0 0  PandarQT map 100" />
    
        <!-- 0524 AG -->
        <!-- same as mapExtrinsicTrans, transform map lime marker from data parser to nearly zero  -->
    <!-- <node pkg="tf" type="static_transform_publisher" name="camera_base_link"
                        args="146 -64  0.0 
                        0.0 0.0 0.0 1.0  map AGmap 100" />
    <node pkg="tf" type="static_transform_publisher" name="PandarQT_map" args="-146 64 0 0 0 0  PandarQT map 100" /> -->
    

        <!-- FIXME: 这里要改，因为这个包已经被re-implement了 -->
    <node name="data_process" pkg="areaGraphDataParser" type="data_process" output="screen"  />
    <node name="CloudHandler" pkg="localizationUsingAreaGraph" type="localizationUsingAreaGraph_cloudHandler"  output="screen" />
    <!-- <node name="particlesGeneratorNode" pkg="localizationUsingAreaGraph" type="generateParticles.py"   output="screen" /> -->

    
    <!-- <node pkg="localizationUsingAreaGraph" type="sendData.py" name="initDataSender" output="screen" /> -->


    <!-- node pkg="rosbag" type="play" name="player" output="screen" args="-q /home/xiefujing/research/AreaGraph/ws/2022-07-03-15-42-01.bag"  / -->
    <!-- <node name="mapHandlerNode" pkg="localizationUsingAreaGraph" type="mapHandler.py"  output="screen" cwd="node"  /> -->
   <!-- <node pkg="localizationUsingAreaGraph" type="getDestinationFromRVIZ.py" name="getDestinationFromRVIZ" output="screen" /> -->
    <!-- <node name="CloudHandler" pkg="localizationUsingAreaGraph" type="localizationUsingAreaGraph_cloudHandler"  output="screen" launch-prefix="xterm -e gdb -ex -args" /> -->


</launch>